site/2017/12/21/orm-2-6-0-and-next.rst

```diff
@@ -0,0 +1,30 @@
+Doctrine ORM 2.6 and Next (3.0)
+=============================================
+
+We are happy to announce the immediate availability of Doctrine ORM 2.6
+`2.6.0 <https://github.com/doctrine/doctrine2/releases/tag/2.6.0>`.
+
+ORM 2.6.0
+---------
+
+This release contains almost 3 years of active development and it provides several improvements and fixes, including:
+
+- Better commit order calculation
+- More stable second level cache
+- Strict testing and quality control
+- PHP 7.1+ requirement
+
+`changelog <https://github.com/doctrine/doctrine2/releases/tag/v2.6.0>`_.
+
+Next ?
+------
+
+This release mark the feature freeze of the 2.x version.
+
+If you need to improve please work on the develop branch that will become the 3.0 version.
```

Comment:
Can we avoid referencing to `develop`? It will be merged into master in upcoming days and cease to exist.

Reply:
Well it doesn't exists yet...